src/signals/settings/users/containers/Overview/index.js

```diff
@@ -103,6 +136,13 @@ export const UsersOverviewContainer = ({ pageSize, userCan }) => {
           <Column span={12}>
             <DataView
               headers={columnHeaders}
+              filters={[
+                (<StyledSearchbar
+                  placeholder=""
```

Comment:
For the ticket regarding the untrimmed input; the `Searchbar` component can receive a value for its `value` prop. You can trim the input, store it in the state and use that to populate the `value` prop.

Reply:
Tried that, doesn't seem to work out of the box...